sdk/cmake/config.cmake

```
@@ -119,3 +119,40 @@ endif()
 
 set(USE_DUMMY_PSEH FALSE CACHE BOOL
 "Whether to disable PSEH support.")
+
+set(TARGET_VERSION XP CACHE STRING "The Windows platform version to target. Valid values are win10 win8.1 win8 win7 vista xp. Defaults to xp.")
```

Maybe commas between values?
```suggestion
set(TARGET_VERSION XP CACHE STRING "The Windows platform version to target. Valid values are win10, win8.1, win8, win7, vista, xp. Defaults to xp.")
```